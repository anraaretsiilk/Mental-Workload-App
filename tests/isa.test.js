import React from 'react';
import renderer from 'react-test-renderer';
import ISA from '../screens/isa';

test('renders isa correctly', () => {
  const tree = renderer.create(<ISA />).toJSON();
  expect(tree).toMatchSnapshot();
});