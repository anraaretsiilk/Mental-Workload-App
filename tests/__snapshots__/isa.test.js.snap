// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders isa correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 20,
    }
  }
>
  <Text>
    ISA
  </Text>
</View>
`;
